DemoApp/Product.cs

```diff
@@ -3,4 +3,5 @@
 public class Product
 {
     public string Name { get; set; }    
+    public string Description { get; set; }
```

Comment:
You do not neet name this property as Description. Just write the **ItemName**

Reply:
Okay